specs/signing-and-verification-workflow.md

```diff
@@ -16,11 +16,11 @@ The user wants to sign an OCI artifact and push the signature to a repository.
     1. Verify that the signing certificate is valid and satisfies [certificate requirements](./signature-specification.md#certificate-requirements).
     1. Verify that the signing algorithm satisfies [algorithm requirements](./signature-specification.md#signature-algorithm-requirements).
     1. Generate signature.
-        1. Generate signature using signature formats specified in [supported signature envelopes](./signature-specification.md#supported-signature-envelopes). Also, as part of this step, the user-defined/supplied custom attributes should be added to the annotations of the signature's descriptor.
-        1. If the user wants to timestamp the signature, obtain an [RFC-3161](https://datatracker.ietf.org/doc/html/rfc3161.html) compliant timestamp for the signature generated in the previous step. Otherwise, continue to the next step.
+        1. Generate signature envelope using signature formats specified in [supported signature envelopes](./signature-specification.md#supported-signature-envelopes). Also, as part of this step, the user-defined/supplied custom attributes should be added to the annotations of the signature's descriptor.
+        1. If the user wants to use timestamp and is signing under signing scheme [`notary.x509`](./signing-scheme.md/#notaryx509), extract the [primitive signature](./signature-specification.md#signature-envelope) (the digital signature computed on payload and signed attributes) from the signature envelope generated in the previous step. Send it to a [RFC-3161](https://datatracker.ietf.org/doc/html/rfc3161.html) compliant `Timestamp Authority (TSA)` for timestamping. Otherwise, continue to the next step. 
             1. Verify that the timestamp signing certificate satisfies [certificate requirements](./signature-specification.md#certificate-requirements).
             1. Verify that the timestamp signing algorithm satisfies [algorithm requirements](./signature-specification.md#signature-algorithm-requirements).
```

Comment:
Those 2 links doesn't work.

Reply:
hmm, the link is correct. I think it's because this is still a PR, once it's merged, the link should be good. By clicking the link in the PR, it redirects to `tsa/specs/...` which does not exist (the branch's name is `tsa`).